src/App.js

```diff
@@ -317,24 +317,22 @@ const returnDefaultComponent = (Component, props) => {
     } else {
       return (
         <div className='component-div h-100'>
-          <Navigation {...props}/>
+          <Navigation />
           <Component {...props} />
           <RequestErrorModal />
         </div>
       );
     }
   } else {
-    return <WalletVersionError {...props} />;
+    return <WalletVersionError />;
   }
 }
 
 /*
  * Return a component with the navigation component
  */
 const NavigationRoute = ({ component: Component, ...rest }) => (
-  <Route {...rest} render={(props) => (
-      returnDefaultComponent(Component, props)
-  )} />
+  returnDefaultComponent(Component, rest)
```

Comment:
The locked screen no longer needs props, so this is a viable simplification.

Reply:
The `Route` wrapper/component adds no logic to work with the react router?

Reply:
Not for a functional component, at least.
The `locked` screen is the only one called by this `NavigationRoute`. I even thought of removing it altogether, but decided to wait until a dedicated PR for the root element.